getting-started/blazor/rendering-different-charts-using-blazor.md

```diff
@@ -63,7 +59,7 @@ const dataset = [
 
 ```
 
-Now that the data is ready, let us dive in directly to render the chart. The consolidated code is given below:
+Now that the data is ready let us dive in directly to render the chart. The consolidated code is given below:
 
 ```javascript
```

Comment:
the cases where we are sharing the entire, we will have to share the entire file's content from the 'examples' directory. Otherwise, user will not be able to run the chart. Because what we have now is body of the code, but the head and footer bits are missing.